js/src/dropdown.js

```diff
@@ -207,6 +207,10 @@ class Dropdown extends BaseComponent {
     this._element.setAttribute('aria-expanded', 'false')
     Manipulator.removeDataAttribute(this._menu, 'popper')
     EventHandler.trigger(this._element, EVENT_HIDDEN, relatedTarget)
+
+    if (this._menu?.contains(document.activeElement)) {
```

Comment:
Is the optional chaining here and in the next line really needed?

Last time I checked (a long time ago) the transpiled code was much bigger. So, if these can be null or undefined maybe just be explicit for now.